ee/fleet-agent-downloader/api/hooks/sso/index.js

```
@@ -47,54 +47,54 @@ module.exports = function (sails){
         'compress',
         'poweredBy',
         'www',// Note: This changes the conventions of Sails.js. Don't ever replicate this or use Passport.js.
-        'oktaSSO',
+        'sso',
```

ESLint

[error] Replace `'sso'` with `"sso"`

(prettier/prettier)